### PR TITLE
Mm feature/fix nested lists in docs

### PR DESCRIPTION
**Pull request by mikemorency**
Updating the format of all of the role READMEs to be consistent. Also fixing nested lists in the README so they display properly in automation hub

### CODE REVIEW

**Review by bardielle — APPROVED** [review]

File: roles/system_settings/README.md [hunk 1]
```diff
@@ -10,103 +10,114 @@ N/A
...
-    * url: Required. Define the URL of the proxy server (including protocol ie. http://...).
-    * port: Required. Define the port of the proxy server.
-    * protocol: Required. Define the protocol of the proxy server(FTP, HTTP, HTTPS).
-    * state: Define if the proxy configuration should be `present` or `absent`.
-    * username: Define `username` for the proxy server if proxy requires authentication.
-    * password: Define `password` for the proxy server if proxy requires authentication.
+- **system_settings_proxy** (list(dict))
+    - A list of proxy configurations.
+    - **Elements**
+        - enabled: bool, required, Define if this proxy configuration should be enabled.
```
Comment (anna-savina) [inline roles/system_settings/README.md:87]:
```suggestion
        - enabled - bool, required, Define if this proxy configuration should be enabled.
```

File: roles/provision_virtual_esxi/README.md [hunk 2]
```diff
@@ -11,82 +11,85 @@ When deploying a new VM:
...
 
-- **provision_virtual_esxi_datacenter**:
-  - str, The name of the datacenter in which you want to deploy the new vms.
+- **provision_virtual_esxi_cluster** (str)
+    - The name of the cluster in which you want to deploy the new vms.
 
-- **provision_virtual_esxi_resource_pool**:
-  - str, The name of the resource pool in which to place the VMs. It's recommended to use a resource pool and limit the amount of resources these ESXi hosts can use
+- **provision_virtual_esxi_datacenter** (str)
+    - The name of the datacenter in which you want to deploy the new vms.
```
Comment (anna-savina) [inline roles/provision_virtual_esxi/README.md:48]:
```suggestion
    - The name of the datacenter in which you want to deploy the new VMs.
```

File: roles/provision_virtual_esxi/README.md [hunk 3]
```diff
@@ -11,82 +11,85 @@ When deploying a new VM:
...
+    - See the [authentication documentation](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/docs/authentication.md) for examples.
 
 ### Placement
-- **provision_virtual_esxi_cluster**:
-  - str, The name of the cluster in which you want to deploy the new vms.
 
-- **provision_virtual_esxi_datacenter**:
-  - str, The name of the datacenter in which you want to deploy the new vms.
+- **provision_virtual_esxi_cluster** (str)
+    - The name of the cluster in which you want to deploy the new vms.
```
Comment (anna-savina) [inline roles/provision_virtual_esxi/README.md:45]:
```suggestion
    - The name of the cluster in which you want to deploy the new VMs.
```

File: roles/info/README.md [hunk 4]
```diff
@@ -118,50 +123,61 @@ An abbreviated example of the data returned can be found below:
...
 
-- **info_appliance_file**
-  - File where to store the gathered data. Default is `/tmp/vmware_ops_info_appliance`
-  - If set to an empty string or `false`, the data is not written to a file.
+- **info_appliance_gather** (list(str))
+    - Define the sections of the appliance to gather. By default we gather all appliance information.
+
+- **info_appliance_file** (str)
+    - File where to store the gathered data. Default is `/tmp/vmware_ops_info_appliance`
+    - If set to an empty string or `false`, the data is not written to a file.
```
Comment (anna-savina) [inline roles/info/README.md:136]:
```suggestion
    - If set to an empty string, the data is not written to a file.
```

Comment (anna-savina) [reply roles/info/README.md:136]: Suggestion is relevant for all `info_xxx_file` vars

File: roles/info/README.md [hunk 5]
```diff
@@ -10,35 +10,40 @@ N/A
...
-  - The variable and it's attributes (license, appliance, etc) are returned regardless if data was gathered or not. However, if you do not gather information pertaining to an attribute an empty data set is returned.
-  - The empty data set will be the same type as a populated dataset; meaning if the data set is normally a list, an empty list will be returned.
-  - For example, if `info_appliance` is set to `false`, then `vmware_ops_info_outputs.appliance` will be `{}`. If `info_guests` is set to `false`, then `vmware_ops_info_outputs.guests` will be `[]`.
+- **info_expose_outputs_as_variable** (bool)
+    - If set to true, the role will expose the gathered information as a variable which can be used later on in your playbook. The variable is called `vmware_ops_info_outputs`. If set to false, this variable is not set.
+    - The variable and it's attributes (license, appliance, etc) are returned regardless if data was gathered or not. However, if you do not gather information pertaining to an attribute an empty data set is returned.
```
Comment (anna-savina) [inline roles/info/README.md:41]:
```suggestion
    - The variable and its attributes (license, appliance, etc) are returned regardless if data was gathered or not. However, if you do not gather information pertaining to an attribute an empty data set is returned.
```

File: roles/system_settings/README.md (hunk 1 above)
Comment (mikemorency) [reply roles/system_settings/README.md:87]: i noticed that other 'elements' sections use the format
\- `varname`: 

so i updated this section to match that format

File: roles/provision_vm/README.md [hunk 6]
```diff
@@ -296,236 +174,79 @@ N/A
...
-        - choices: ['disabled', 'opportunistic', 'required']
-
-- **provision_vm_force** (boolean):
+    - **Elements**
+        - `encrypted_vmotion`: str, Controls encryption for live migrations with vmotion. Choices: [`disabled`, `opportunistic`, `required`]
+        - `encrypted_ft`: str, Controls encryption for fault tolerance replication. Choices: [`disabled`, `opportunistic`, `required`]
+
+- **provision_vm_force** (bool)
     - Ignore warnings and complete the actions.
     - This parameter is useful while removing a virtual machine that is the powered on state.
```
Comment (anna-savina) [inline roles/provision_vm/README.md:212]:
```suggestion
    - This parameter is useful while removing a virtual machine that is in the powered on state.
```

File: roles/provision_vm/README.md [hunk 7]
```diff
@@ -296,236 +174,79 @@ N/A
...
     - If true the virtual machine's disks will not be deleted when `provision_vm_state` is `absent`, but the VM will still be removed from the vSphere inventory.
-    - Choices:
-        - false (default)
-        - true
 
-- **provision_vm_disk** (list):
+- **provision_vm_disk** (list)
     - A list of disks to add.
     - This parameter is case sensitive.
     - Shrinking disks is not supported.
```
Comment (anna-savina) [inline roles/provision_vm/README.md:183]:
```suggestion
    - Shrinking disks are not supported.
```

File: roles/provision_vm/README.md [hunk 8]
```diff
@@ -296,236 +174,79 @@ N/A
     - Incorrect key and values will be ignored.
```
Comment (anna-savina) [inline roles/provision_vm/README.md:174]:
```suggestion
    - Incorrect keys and values will be ignored.
```

File: roles/provision_vm/README.md [hunk 9]
```diff
@@ -12,280 +12,158 @@ N/A
...
+    - Choices: [`true`, `false`]
 
-- **provision_vm_snapshot_src** (string):
+- **provision_vm_snapshot_src** (str)
     - Name of the existing snapshot to clone and use to create a new virtual machine.
     - This parameter is case sensitive.
     - This parameter is required when using the `provision_vm_linked_clone` parameter.
 
-- **provision_vm_advanced_settings** (list):
+- **provision_vm_advanced_settings** (list)
```
Comment (anna-savina) [inline roles/provision_vm/README.md:120]:
```suggestion
- **provision_vm_advanced_settings** (list(dict))
```

File: roles/provision_vm/README.md [hunk 10]
```diff
@@ -296,236 +174,79 @@ N/A
...
 
-- **provision_vm_delete_from_inventory** (boolean):
+- **provision_vm_delete_from_inventory** (bool)
     - If true the virtual machine's disks will not be deleted when `provision_vm_state` is `absent`, but the VM will still be removed from the vSphere inventory.
-    - Choices:
-        - false (default)
-        - true
 
-- **provision_vm_disk** (list):
+- **provision_vm_disk** (list)
```
Comment (anna-savina) [inline roles/provision_vm/README.md:180]:
```suggestion
- **provision_vm_disk** (list(dict))
```

File: roles/provision_vm/README.md [hunk 11]
```diff
@@ -536,219 +257,97 @@ N/A
...
+    - Choices: [`absent`, `poweredon`, `powered-on`, `poweredoff`, `powered-off`, `present`, `rebootguest`, `reboot-guest`, `restarted`, `suspended`, `shutdownguest`, `shutdown-guest`]
+
+- **provision_vm_state_change_timeout** (int)
     - If `provision_vm_state` is `shutdownguest`, the module will return immediately after sending the shutdown signal.
     - If this argument is set to a positive integer, the module will wait for the virtual machine to reach the poweredoff state.
-    - Default: 0
+    - Default is 0
 
-- **provision_vm_vapp_properties** (list):
+- **provision_vm_vapp_properties** (list)
```
Comment (anna-savina) [inline roles/provision_vm/README.md:267]:
```suggestion
- **provision_vm_vapp_properties** (list(dict))
```

File: roles/provision_vm/README.md [hunk 12]
```diff
@@ -536,219 +257,97 @@ N/A
...
 
-- **provision_vm_wait_for_ip_address_timeout** (integer):
+- **provision_vm_wait_for_ip_address_timeout** (int)
     - Define a timeout (in seconds) for the `provision_vm_wait_for_ip_address` parameter.
-    - Default: 300
+    - Default is 300
 
 
-- **provision_vm_networks** (list):
+- **provision_vm_networks** (list)
```
Comment (anna-savina) [inline roles/provision_vm/README.md:297]:
```suggestion
- **provision_vm_networks** (list(dict))
```

**Review by anna-savina — APPROVED** [review]
LGTM

**Pull request merged**